src/components/smartSearch/inputs/StyledTagSelect.tsx

```diff
@@ -0,0 +1,52 @@
+import { Theme } from '@material-ui/core';
+import { Autocomplete, AutocompleteProps } from '@material-ui/lab';
+import { Chip, makeStyles } from '@material-ui/core';
+
+import StyledTextInput from './StyledTextInput';
+import { ZetkinTag } from 'types/zetkin';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+    autocomplete: {
+        display: 'inline',
+    },
+    input: {
+        lineHeight: theme.typography.h4.lineHeight,
+        verticalAlign: 'middle',
+        width: 'auto',
+    },
+}));
+
+type StyledTagSelectProps = Omit<AutocompleteProps<ZetkinTag, true, undefined, false>, 'renderInput'>
+
+const StyledTagSelect = (props: StyledTagSelectProps): JSX.Element => {
```

Comment:
How do you feel about the "clear" button on this? Is it possible to remove it? I think the UI is a little bit busy:

![image](https://user-images.githubusercontent.com/550212/128150506-ab313b89-8025-4c88-b2cc-01dae7bf0b44.png)

* Tag 1 name
* X to remove tag 1
* Tag 2 name
* X to remove tag 2
* Input text
* X to remove all tags
* Arrow to toggle select list

I foresee someone accidentally clicking that "x" that clears all the tags and having to restart. It's very rare that you have added a ton of tags and need to remove them all at once.

So a way to avoid that, and "clean up" the UI a little bit at the same time would be to just hide the "clear" X button.